### DISCUSSION TITLE
I canceled copilot free try but want to pay now

### DISCUSSION THREAD

**Discussion by GuoJing** [community/community#29920] [Copilot Conversations]
Hi

I canceled the copilot free try, but after 1-month of usage, I think this is ok, so I want to pay now but I can't find a place to pay the bill.

Where to pay the bill?

:)

<img width="803" alt="WechatIMG14600" src="https://user-images.githubusercontent.com/363073/185365240-551b4ce7-fa94-4aef-9369-88b5a2314677.png">

**Comment by iamcjwilliams** [comment]
Thanks for reaching out about this, we're sorry for the trouble here. Our GitHub Support team will be glad to help if you please reach out to them directly using our [secure Support form](https://support.github.com/). Please note that due to current high volume, there may be a longer than usual wait time :)
